src/main/clojure/clojure/math/combinatorics.cljc

```diff
@@ -9,194 +9,188 @@
      :doc "Efficient, functional algorithms for generating lazy
 sequences for common combinatorial functions. (See the source code 
 for a longer description.)"}
-  clojure.math.combinatorics
-  (:refer-clojure :exclude [update]))
+  clojure.math.combinatorics
+  (:refer-clojure :exclude [update])
+  #?(:cljs (:require [goog.string :refer [format]])))

```

Comment:
@jurgenphotek the correct way of using `goog.string.format` is by requiring `goog.string.format` like this: `(:require goog.string.format)`